docs/experiment_tracking.md

```diff
@@ -0,0 +1,88 @@
+# Experiment tracking in llama recipes. 
+
+Llama recipes has an optional support for tracking experiments using experiment tracker modules like Aimstack or WandB (Currently not supported).
+
+The experiment tracker can be enabled by passing an additional argument to the llama recipes code which loads and
+tracks all the training and evaluation metrics in the tracker along with the system metrics being tracker by these trackers.
+
+The addition argument can be passed as - 
+
+```
+$ torchrun --nnodes 1 --nproc_per_node 1 examples/finetuning.py \
+           --model_name MODEL_NAME --output_dir OUTPUT_DIR \
+           --use_peft --peft_method lora --pure_bf16 \
+           --tracker aim
+```
+
+List of trackers integrated into llama recipes along with their support instructions are below.
+
+## Aimstack (https://aimstack.io/)
+
+Current version of the code supports Aimstack tracking via the trackers module.
+
+One needs to install Aimstack separately if need to use aimstack as a tracker.
+
+### Installation
+
+Aim can be installed by running
+
+```
+$ pip install aim==3.17.5
```

Comment:
do we require a specific version or can we put `>=` ?

Reply:
Aimstack sdk is changing across versions so I am putting the version which I have found to be stable.